articles/rules/metadata-in-rules.md

```diff
@@ -310,3 +310,43 @@ The resulting user is:
   }
 }
 ```
+
+### Considerations
+
+Metadata key field names in `user_metadata` or `app_metadata` with a dot are not allowed by the internal storage mechanism used by Auth0.
```

Comment:
Rather than say "internal storage" say the app metadata must be a valid JSON object and can not contain a dot in property names. We should never discuss "internal" implementations as a reason because the internal mechanism may change, but our schema won't.